NOTICE

```diff
@@ -1,3 +1,4 @@
 Kubernetes Anywhere
 Copyright 2015-2016 Weaveworks Ltd.
```

Comment:
Did you intend to leave this one? I think you are included on "Kubernetes Authors" umbrella. cc @bgrant0607